include/openthread/platform/settings.h

```diff
@@ -80,10 +80,17 @@ enum
 /**
  * Performs any initialization for the settings subsystem, if necessary.
  *
- * @param[in]  aInstance The OpenThread instance structure.
+ * This function also sets the critical keys that should be stored in the secure area.
+ *
+ * Note that the memory pointed by @p aCriticalKeys MUST not be released before @p aInstance is destroyed.
+ *
+ * @param[in]  aInstance            The OpenThread instance structure.
+ * @param[in]  aCriticalKeys        A pointer to an array containing the list of critical keys. May be NULL only if
+ *                                  @p aCriticalKeysLength is 0, which means that there is no critical keys.
+ * @param[in]  aCriticalKeysLength  The number of entries in the @p aCriticalKeys array.
  *
  */
-void otPlatSettingsInit(otInstance *aInstance);
+void otPlatSettingsInit(otInstance *aInstance, const uint16_t *aCriticalKeys, uint16_t aCriticalKeysLength);
```

Comment:
nit: how about renaming to `aSensitiveKeys`? I think "sensitive" is more descriptive than "critical".

Reply:
Done.